gamemodes/terrortown/gamemode/shared/lang/polish.lua

```diff
@@ -0,0 +1,1222 @@
+-- English language strings

```

Comment:
you should change this to "polish"

Reply:
Doing it in a minute

Reply:
Done, I think translating --lines is pointless, because no one will ever see it in game, so error logs are also in English

Reply:
yes, comments should be always in english :)